tests/profiling/collector/test_stack.py

```diff
@@ -73,7 +73,7 @@ def test_collect_once():
         if e.thread_name == "MainThread":
             if TESTING_GEVENT:
                 assert e.task_id > 0
-                assert e.task_name == e.thread_name
+                # assert e.task_name == e.thread_name  # This can actually be another coroutine like the tracer writer
```

Comment:
should we change to `assert e.task_name is not None` ?

the `else` case here asserts that it is `None`.

Reply:
Good point.